lib/tasks/rubocop.rake

```diff
@@ -5,5 +5,5 @@ begin
     task.options = %w(--format simple)
   end
 rescue LoadError
-  puts 'Unable to load RuboCop.'
+  puts 'Unable to load RuboCop.' if Rails.env.development? || Rails.env.test?
```

Comment:
argo-dev  runs in development mode, so if we deploy to argo-dev, will we also get bombarded with these messages?